components/display/index.rst

```diff
@@ -14,8 +14,18 @@ engine. Fundamentally, there are these types of displays:
 - Graphical binary displays which can toggle ON/OFF any pixel, like :doc:`E-Paper <waveshare_epaper>`,
```

Comment:
When i read "binary displays", i see this as monochrome displays. Could we split this into *binary* and *color* displays? 

I know this has noting to do with LVGL, but i see it now. And i did not realize before it was there.

Reply:
Binary as binary data, I guess. As contrary to Character displays.

Reply:
This caught me a bit off-guard, too. I think it makes sense, but e-paper displays aren't the first thing I think of when I hear "binary display". haha

Reply:
Maybe we could just forget the word binary...

Reply:
Yes go for it :D